scriptmodules/supplementary/runcommand.sh

```diff
@@ -19,7 +19,7 @@ function _update_hook_runcommand() {
 }
 
 function depends_runcommand() {
-    local depends=()
+    local depends=(python-pyudev)
```

Comment:
For now I'll remove it from runcommand.sh and put in system.sh. And later we can check how the module does to implement it directly. Are you OK with it?